processor/servicegraphprocessor/README.md

```diff
@@ -117,15 +117,16 @@ The following settings are required:
 
 The following settings can be optionally configured:
 
-- `store` defines the config for the in-memory store used to find requests between services by pairing spans.
```

Comment:
I was thinking of removing the entire contents of the processor's configuration section and directly linking to the connector's section, so we don't have a copy of information that we have to maintain. My concern though is there's a possibility of adding connector-specific options at some point, and also since the processor is deprecated maintaining these duplicates may not be a long term issue. 

Happy to use a link instead though if others think that's preferable.